modules/beacon/mkosi.extra/usr/local/lib/scripts/beacon-report.sh

```diff
@@ -29,7 +29,12 @@ for sev_version in "${SEV_VERSIONS[@]}"; do
   SEV_CERT_FILE="${HOME:-/root}/sev_certificate_v${sev_version}.txt"
 
   # Call beacon
-  beacon report --title "$SEV_TITLE" --body "$SEV_CERT_FILE" --label "certificate" --label "${OS_LABEL}"
+  if [ -e "${SEV_CERT_FILE}" ] && [ -z "$(grep "❌" "${SEV_CERT_FILE}")" ]; then
+    # Add milestone if no errors encountered
```

Comment:
So, that''s not how the certificate generation is supposed to work. We might need broader changes here if it doesn't meet the design criteria. Here's how it is supposed to work:

1. Every test has an associated version number.
2. We run all tests.
3. We certify a distro with the newest version number that has no failed tests in equal or later versions.

For example, if a distro runs the following tests:
* foo (v3.1-0) PASS
* bar (v3.0-8) FAIL
* baz (v3.0-4) PASS

We would certify the distro only for `v3.0-4`. The version that applies to each test is specify as extra data in the systemd unit file.